### DISCUSSION TITLE
Problem with backspace\delete in text trait

### DISCUSSION THREAD

**Discussion by rlmorgan7** [PhenoApps/Field-Book#782] [Q&A] [answered]
I've just updated to the latest version (v 5.5.16). The only problem that I have found is that when typing on the tablet keyboard in the text trait, the delete and backspace buttons don't work. I'm using a Samsung Tab A7 Lite tablet. Has anyone else seen this problem/found a solution?

**Comment by trife** [comment]
Can you post a screenshot of your Collect screen?

**Comment by rlmorgan7** [comment]
![20231026_134109](https://github.com/PhenoApps/Field-Book/assets/149088060/b03a01d1-45d4-4a28-83a2-11526ae59275)

**Reply by splGH** [reply]
I am having the same problem on my Samsung Lite A7.  The trash can button stopped working to delete a data entry. Also for traits with categorical variables, the category buttons disappear. I was able to restore both functions briefly after restarting several times, but then it very quickly stopped functioning correctly. I have version 5.5.24.

**Comment by trife** [comment, marked as answer]
We're able to reproduce the issue on our Samsung test device and are working on a fix. Thanks for reporting this!

**Reply by rlmorgan7** [reply]
Thanks very much :) The rest of the upgrade (especially the photos) was much appreciated ... and then I ran into this problem.
